src/lib/updateGrantsRecipients.js

```diff
@@ -223,6 +302,15 @@ export async function processFiles(hashSumHex) {
           transaction,
         },
       );
+
+      // Update grant personnel.
+      await ProgramPersonnel.unscoped().bulkCreate(
```

Comment:
Is there a reason to use unscoped here? If we ever decide to add a default scope or something this may cause some annoyance

Reply:
I was following the same pattern as the other bulk update.

Reply:
The other bulk update is for Grants, which have a default scope.

Reply:
I removed the .unscoped()